spec/yaml/swiftnav/sbp/observation.yaml

```diff
@@ -80,32 +103,57 @@ definitions:
             type: CarrierPhase
             units: cycles
             desc: Carrier phase observation with typical sign convention.
+        - D:
```

Comment:
Why do we need to add measured doppler to this message? Is this for passing a higher-resolution measured doppler to the rover from the base station (as the rover might be using measured vs tdcp doppler)? Is there a sense of the actual upside in not just using tdcp from the base station?